### PR TITLE
Brantley's 9/13 Data Briefing

### PR DESCRIPTION
**Pull request by ToniBonittoGSA**
filename and slug don't match

2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md
the-data-briefing--how-blockchain-s-future-growth-will-affect-federal-agencies

### CODE REVIEW

**Comment by thisisdano** [comment]
And neither of them match the url on digitalgov.gov: 
`the-data-briefing-how-blockchains-future-growth-will-affect-federal-agencies`

@jeremyzilar So I think we need to change the filename/slug builder to do a couple things:
- prevent repeated hyphens `--`
- don't use a hyphen for instances of `'`: just strip the character
- OR, do all special characters get stripped rather than converted to `-`? This might solve both issues...

File: content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+slug: the-data-briefing--how-blockchain-s-future-growth-will-affect-federal-agencies
```
Comment (thisisdano) [inline content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md]: Should match the slug on DG: `the-data-briefing-how-blockchains-future-growth-will-affect-federal-agencies`

**Comment by jeremyzilar** [comment]
I can make sure we don't have double dashes in the filenames and slugs.

Other than that, I think we might just need to check that both posts have consistent URLs before publishing on Federalist

Once we are just on Federalist, this shouldn't be an issue.

File: content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The amount of disruption to an agency’s business processes explains why it may be easier for federal agencies to adopt chatbots than blockchains. Implementing a chatbot can use existing agency business processes and customer service knowledge to build the database for the conversational interface. Implementing a blockchain requires a large-scale organizational change project to prepare the federal agency for the distributed trust environment.
+
+As many of you who have worked on large-scale change projects in your agency know, changing how an organization operates is a Herculean effort. Even though chatbots may be ready now, blockchains are still an adolescent technology with many years of refinement ahead for the technology. The promise of a digital autonomous federal organization may be a decade or two away. Meanwhile, I expect to see exponential growth in federal agency chatbots over the next decade and the rise of more augmentation technologies for federal workers.
+
+HR
```
Comment (thisisdano) [inline content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md]: Use the markdown equivalent of a `<hr/>` here: `---` (three hyphens)

File: content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Each week, The Data Briefing showcases the latest federal data news and trends. Visit this blog every week to learn how data is transforming government and improving government services for the American people. If you have ideas for a topic or have questions about government data, please contact me via email.
+
+Dr. William Brantley is the Training Administrator for the U.S. Patent and Trademark Office’s Global Intellectual Property Academy. All opinions are his own and do not reflect the opinions of the USPTO or GSA.
+
+HR
```
Comment (thisisdano) [inline content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md]: Use the markdown equivalent of a `<hr/>` here: `---` (three hyphens)

**Pull request merged**